packages/shared/src/components/cards/AdList.tsx

```diff
@@ -13,9 +12,7 @@ export const AdList = forwardRef(function AdList(
     <ListCard {...domProps} data-testid="adItem" ref={ref}>
       <AdLink ad={ad} onLinkClick={onLinkClick} />
       <ListCardMain>
-        <ListCardTitle
-          className={classNames('line-clamp-4 font-bold typo-title3')}
-        >
+        <ListCardTitle className="font-bold line-clamp-4 typo-title3">
```

Comment:
I was checking through implementations of `line-clamp-4` and just noticed it is using `classnames` unnecessarily. Nothing really changed here.